source/index.html.md

```diff
@@ -1177,7 +1300,18 @@ organization_id_eq | Search for users in a specific organization
 
 `PATCH /v3/users/:id`
 
-## Delete a User
+## Delete/Deactivate a User
+You can **soft-delete** a user to deactivate their account (they will no longer be able to log in) but still see their appointment history.
+
+Common scenarios for using this feature are:
+* If an employee is fired
```

Comment:
make this language a little softer. "an employee is no longer with the company" or "let go" or something